### PR TITLE
[Validator] Add a requireTld option to Url constraint

### LINKED ISSUE #19746: [Validator] Add a `requireTld` option to `Url` constraint
| Q            | A
| ------------ | ---
| Feature PR   | symfony/symfony#54408
| PR author(s) | @javiereguiluz
| Merged in    | 7.1

We created this issue to not forget to document this new feature. We would really appreciate if you can help us with this task. If you are not sure how to do it, please ask us and we will help you.

To fix this issue, please create a PR against the 7.1 branch in the [symfony-docs repository](https://github.com/symfony/symfony-docs).

Thank you! :smiley:

### PR DESCRIPTION
**Pull request by javiereguiluz**
Fixes #19746.

### CODE REVIEW

File: reference/constraints/Url.rst [hunk 1]
```diff
@@ -307,3 +307,109 @@ also relative URLs that contain no protocol (e.g. ``//example.com``).
...
+.. note::
+
+    This constraint does not validate that the given TLD value is included in
+    the `list of official top-level domains`_ (because that list is growing
+    continuously and it's hard to keep track of it).
+
+``tldMessage``
+~~~~~~~~~~~~~~
+
+**type**: ``string`` **default**: ``TThis URL does not contain a TLD.``
```
Comment (OskarStark) [inline reference/constraints/Url.rst]:
```suggestion
**type**: ``string`` **default**: ``This URL does not contain a TLD.``
```

File: reference/constraints/Url.rst [hunk 2]
```diff
@@ -307,3 +307,109 @@ also relative URLs that contain no protocol (e.g. ``//example.com``).
...
+
+        // src/Entity/Website.php
+        namespace App\Entity;
+
+        use Symfony\Component\Validator\Constraints as Assert;
+
+        class Website
+        {
+            #[Assert\Url(
+                tldMessage: 'Add at least one TLD to the {{ value }} URL.',
```
Comment (OskarStark) [inline reference/constraints/Url.rst:365]: This is a bit confusing, isn't it, as `requireTld` option default is `false`, setting only the message here will have no effect.

File: reference/constraints/Url.rst (hunk 1 above)
Comment (javiereguiluz) [reply reference/constraints/Url.rst]: Fixed. Thanks!

File: reference/constraints/Url.rst (hunk 2 above)
Comment (javiereguiluz) [reply reference/constraints/Url.rst:365]: You are right. Thanks for the review. It's fixed now.

**Pull request merged**